app/controllers/shared/api.php

```diff
@@ -511,8 +518,15 @@
             ->setIP($request->getIP())
             ->setHostname($request->getHostname())
             ->setEvent($route->getLabel('audits.event', ''))
-            ->setProject($project)
-            ->setUser($user);
+            ->setProject($project);
+
+        /* If a session exists, use the user associated with the session */
+        if (!$user->isEmpty()) {
+            $typedUser = clone $user;
```

Comment:
```suggestion
            $userClone = clone $user;
```
please update the var name everywhere